provisioner/powershell/provisioner.go

```diff
@@ -116,15 +116,15 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 	}
 
 	if p.config.ElevatedEnvVarFormat == "" {
-		p.config.ElevatedEnvVarFormat = `$env:%s="%s"; `
+		p.config.ElevatedEnvVarFormat = `$env:%s=\"%s\"; `
 	}
 
 	if p.config.ExecuteCommand == "" {
-		p.config.ExecuteCommand = `powershell "& { {{.Vars}}{{.Path}}; exit $LastExitCode}"`
+		p.config.ExecuteCommand = `powershell '& {$ProgressPreference=\"SilentlyContinue\"; {{.Vars}}{{.Path}}; exit $LastExitCode}'`
```

Comment:
I also have the same problem.
environment_vars are not being passed into the powershell provisioner.
I understand that LastExitCode will display the error of the powershell script not the failure to declare the environment_vars. 

In my case the Powershell script is failing because those environment_vars are not present.